### PR TITLE
Resolves #46 include user id in learnerinfo as per adapt-offlineStorage-scorm.js

### LINKED ISSUE #46: OfflineStorage does not store the student id
The code at https://github.com/adaptlearning/adapt-contrib-xapi/blob/master/js/adapt-offlineStorage-xapi.js#L63-L82 does not set the `id` property.  I think we should be consistent with the Spoor plugin in this regard.

### PR DESCRIPTION
**Pull request by danielstorey**

### CODE REVIEW

File: js/adapt-offlineStorage-xapi.js [hunk 1]
```diff
@@ -58,11 +58,12 @@ define([
     },
 
     /**
-     * @returns {{name: string in the format Firstname Lastname, firstname: string, lastname: string }}
+     * @returns {{id: string, name: string in the format Firstname Lastname, firstname: string, lastname: string }}
```
Comment (moloko) [inline js/adapt-offlineStorage-xapi.js]:
```suggestion
     * @returns {{id: string, name: string, firstname: string, lastname: string}} The learner's id (Actor.account.name), full name (in the format Firstname Lastname), first and last names
```

In VSC, that generates the following:
2019-02-14 11_37_29-adapt-offlinestorage-xapi js - dev - visual studio code

File: js/adapt-offlineStorage-xapi.js [hunk 2]
```diff
@@ -58,11 +58,12 @@ define([
...
      */
     getLearnerInfo: function() {
       var actor = xapi.get('actor') || {};
       var name = actor.name || '';
+      var id = actor.account && actor.account.name;
```
Comment (brian-learningpool) [inline js/adapt-offlineStorage-xapi.js]: Maybe the `id` should be something like `mbox`, `openid` or we could add a new config setting to the extension to let the user specify?

**Comment by danielstorey** [comment]
@brian-learningpool raised some additional points, post approval,  re falling back on openid and mbox properties

File: js/adapt-offlineStorage-xapi.js [hunk 3]
```diff
@@ -58,7 +58,7 @@ define([
     },
 
     /**
-     * @returns {{name: string in the format Firstname Lastname, firstname: string, lastname: string }}
+     * @returns {{id: string, name: string, firstname: string, lastname: string}} The learner's id (Actor.account.name), full name (in the format Firstname Lastname), first and last names
```
Comment (moloko) [inline js/adapt-offlineStorage-xapi.js]: comment needs updating following f1aa6d6

```suggestion
     * @returns {{id: string, name: string, firstname: string, lastname: string}} The learner's id, full name (in the format Firstname Lastname), first and last names
```

**Pull request merged**